lib/logstash/codecs/identity_map_codec.rb

```diff
@@ -164,6 +164,16 @@ def encode(event, identity = nil)
     stream_codec(identity).encode(event)
   end
 
+  def flush_mapped(listener)
```

Comment:
Its used in the file input (or any input that wraps the given codec in an IMC)
Needs a different PR for file input.